### ISSUE TITLE
Yolov3 loss

### ISSUE THREAD

**Issue by rjtshrm** [eriklindernoren/PyTorch-YOLOv3#743]
I have a question regarding yolov3 loss. Suppose for a feature location at all 3 scales, a particular location is only associated with one object's center point. So should we associate this object with all the anchors that has best Iou at different scale to calculate loss function or the one anchors over all other anchors at all scales which has max IoU with the object and discard other anchors. In the first case, at different scales particular feature location will backpropagate the loss for this object where as in the later case only one scale which has best overall IoU with this object is responsible for the loss. Any insight on this would be helful.

**Comment by Flova** [comment]
The object is added to the targets at all output layer scales at the corresponding anchor and possible duplicate predictions are reduced by the nms afterward if I recall correctly. Does this answer your question?

**Comment by rjtshrm** [comment]
Yes you answered it, and thanks.
